### ISSUE TITLE
Does it work with scoped generators?

### ISSUE THREAD

**Issue by chakkaradeep** [SamVerschueren/vscode-yo#48]
Hi there,

The [SharePoint Yeoman Generator](https://www.npmjs.com/package/@microsoft/generator-sharepoint) to scaffold web part projects is currently into a npm scope, which means, I will have to run the yo as:

```
yo @microsoft/sharepoint
```

Does the yo extension support this? When I use the extension in VS code, it just shows `sharepoint` as the generator and not `@microsoft/sharepoint`. The next step of showing sub generators also succeeds. But then once I select the default `app` sub-generator, it throws the following error:

```
You don't seem to have a generator with the name "sharepoint:app" installed.
```

The generator as such when used in the command line works without any issues. 

Any help much appreciated.

Thanks,
Chaks

**Comment by SamVerschueren** [comment]
Good point, didn't know it was possible. I will have a closer look when I find some time.

**Comment by chakkaradeep** [comment]
@SamVerschueren Do you have any updates on this? Thanks!

**Comment by SamVerschueren** [comment]
PR #49 should resolve this. I hope to merge it in the coming days. Feel free to check out the PR and try it yourself to see if it works. Feedback is much appreciated!

**Comment by chakkaradeep** [comment]
Do you have instructions on how to test the extension locally? I tried two things: 

I cloned the PR source [repo](https://github.com/zuoez02/vscode-yo) and started debugging. But that results in [errors](http://pastebin.com/f3GcQg29).

I also tried copying the folder into .vscode/extensions (in my mac). This also doesn't work as VS Code says _yo command not found_.

**Comment by SamVerschueren** [comment]
I tested it and it works. BUT, there is a line in `generators/solution/index.js` of the `@microsoft/sharepoint` generator

```js
const process = require('process');
```

This makes this extension fail because `process` already exists (globally). Could you send a PR internally to remove that as that is not used?

**Comment by chakkaradeep** [comment]
@SamVerschueren Thanks! Have forwarded this to my dev team. Lets me work with them to see what is going on. 

If you can update your ReadMe on steps to test/deploy this extension locally, then it would be very helpful for us to validate as well.

**Comment by SamVerschueren** [comment]
@chakkaradeep To test the extension, clone the repository, install the dependencies (`npm install`), open the project in VS Code and in the debug section, just click run.

**Comment by SamVerschueren** [comment]
Released in `0.9.3`. Thanks to @zuoez02 !
